packages/components/atoms/f-error-boundary/src/components/ErrorBoundary.vue

```diff
@@ -0,0 +1,73 @@
+<script>
+import loggerService from '../service/logger.service';
+
+export default {
+    props: {
+        tier: {
+            type: String,
+            default: null
+        },
+
+        stopPropagation: {
+            type: Boolean,
+            default: true
+        },
+
+        hideOnError: {
+            type: Boolean,
+            default: false
+        }
+    },
+
+    data: () => ({
+        hasError: false,
+        error: null,
+        vm: null,
+        info: null
+    }),
+
+    computed: {
+        loggerPayload () {
+            return loggerService.buildVueError(this.error, this.vm, this.info, {
+                tier: this.tier
+            });
+        }
+    },
+
+    errorCaptured (error, vm, info) {
+        return this.interceptError(error, vm, info);
+    },
+
+    methods: {
+        interceptError (error, vm, info) {
+            this.hasError = true;
+            this.error = error;
+            this.vm = vm;
+            this.info = info;
+
+            this.$emit('on-error', {
+                error,
+                vm,
+                info,
+                tier: this.tier,
```

Comment:
Do we need tier to be represented here, when we're also including it as part of the `loggerPayload` object?

Reply:
I included it in case people want to use it for any other purposes, outside of logging.